src/components/Dashboard/Users.js

```diff
@@ -47,14 +48,20 @@ export class Users extends Component {
             {users.map((user, index) => {
               return (
                 <tr key={index}>
-                  <td>{user.id}</td>
+                  <td className="text-center">{user.id}</td>
                   <td>{user.emailAddress}</td>
-                  <td>{user.activationCode === null ? 'Yes' : 'No'}</td>
-                  <td>
+                  <td className="text-center">
+                    {user.activationCode === null ? (
+                      <FontAwesomeIcon icon="check" color="green" title="Yes" />
```

Comment:
It's up to you but one thing I sometimes do with things like this is to split the icons out into separate getter methods and then just `{user.activationCode === null ? this.yesIcon : this.noIcon}`

Reply:
I added this